dashboard/app/views/layouts/_share_footer.html.haml

```diff
@@ -0,0 +1,41 @@
+#page-small-footer
+
+:ruby
+  reactProps = {
+    i18nDropdown: '',
+    copyrightInBase: false,
+    baseMoreMenuString: I18n.t('footer.built_on_code_studio'),
+    className: 'dark',
+    menuItems: [
+      {
+        text: I18n.t('footer.make_my_own_app'),
+        link: 'https://code.org/learn',
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.how_it_works'),
+        link: url_for(action: 'edit'),
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.copyright'),
+        link: '#',
+        copyright: true
+      },
+      {
+        text: I18n.t('footer.tos'),
+        link: "https://code.org/tos",
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.privacy'),
+        link: "https://code.org/privacy",
+        newWindow: true
+      }
+    ],
+    copyrightStrings: build_copyright_strings
+  }
```

Comment:
This logic already existed in a _small_footer layout here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/_small_footer.html.haml

I forked it to _share_footer since they each have a different layout. Are you suggesting removing both of these in favor of a helper or controller?